tools/wine4office-manager/wine4office_backend.py

```
@@ -1185,19 +1201,27 @@ def stop_wine(prefix_value: str, wine_value: str, use_x11: bool = True,
             timeout=max(0.001, deadline - time.monotonic()),
-            check=True,
         )
-        graceful_close = True
+    except RuntimeError:
+        office_pids = []
+    close_command = [str(wine), "wine4officeclose.exe"]
+    for pid in office_pids:
+        close_command.extend(("--pid", str(pid)))
+    try:
+        if office_pids:
+            subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)